diff --git a/ka9q-utils.sh b/ka9q-utils.sh
index 17266b7..2913cd5 100755
--- a/ka9q-utils.sh
+++ b/ka9q-utils.sh
@@ -758,6 +758,12 @@ declare ka9q_service_daemons_list=(
 
 ### This is called by the watchdog daemon and needs to be extended to support multiple RX888 servers at a site.
 function ka9q_web_daemon() {
+
+    if [[ ${KA9Q_RUNS_ONLY_REMOTELY-no} == 'yes' ]]; then
+       wd_logger 1 "KA9Q_RUNS_ONLY_REMOTELY=${KA9Q_RUNS_ONLY_REMOTELY} - So we do not need ka9q_web_daemon() to attempt to start KA9Q-Radio Web as we are using an external / independent instance."
+       return 0
+    fi
+
     wd_logger 1 "Starting"
 
     while true; do
